src/main/resources/static/sasan.html

```diff
@@ -1,3 +1,8 @@
+<link href="https://fonts.googleapis.com/css2?family=Secular+One&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="sasan.css" >
```

Comment:
you need to change css file name here too and also need to change the sasan.html to error.html.

Reply:
Yes I will make all the recommended changes and will also attach the screenshot .Thank you